pytest/tests/sanity/state_sync_then_catchup.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
```

Comment:
nit: It's good practice to wrap the code in a unittest and a _ _ main _ _ like so:
https://github.com/near/nearcore/tree/master/pytest#creating-new-tests

Reply:
Added `main()`, thanks!
Nayduck doesn't expect these tests to be unittests.

Reply:
I think as long as you run unittest.run() from within main it should work fine. Did you try and did it not work? Maybe you also need to put the test logic in a method with name beginning with test_. Not a biggie though, don't worry about it too much.